2.hello-kubernetes/README.md

```diff
@@ -81,7 +81,17 @@ Windows
 for /f "delims=" %a in ('kubectl get svc nodeapp --output 'jsonpath={.status.loadBalancer.ingress[0].ip}') do @set NODE_APP=%a
 ```
 
-## Step 4 - Deploy the Python App with the Dapr Sidecar
+## Step 4 - Verify Service call using external IP 
```

Comment:
Can we have a note for minikube users just like the one in the above step?

Reply:
Isn't that already there in the previous step? Why is there a necessity for repetition?

Reply:
It'll be easy for minikube users to ignore this step when we mention it explicitly else they might be confused why this step fails for them. If not the note, we can just have this:
"## Step 4 - Verify Service call using external IP (minikube users can ignore this step)"